apps/chatbots/forms.py

```diff
@@ -40,6 +40,59 @@ def save(self, commit=True):
         return experiment
 
 
+class ChatbotSettingsForm(forms.ModelForm):
+    description = forms.CharField(widget=forms.Textarea(attrs={"rows": 2}), required=False)
+    seed_message = forms.CharField(widget=forms.Textarea(attrs={"rows": 2}), required=False)
+
+    class Meta:
+        model = Experiment
+        fields = [
+            "description",
+            "voice_provider",
```

Comment:
The querysets for these fields need to be filtered according to the team: e.g. https://github.com/dimagi/open-chat-studio/blob/9fd6e85e77e0408c4d6453050aa01385cdd53305/apps/experiments/forms.py#L186-L198